src/assets/scss/_ngx-datatable.scss

```diff
@@ -2,42 +2,50 @@
 
 .ngx-datatable.bootstrap {
   background-color: $content-block-bg-color;
-
   box-shadow: none;
   font-size: 13px;
+
   .datatable-header {
     height: unset !important;
     background-color: $content-block-bg-color;
+
     .datatable-header-cell {
       vertical-align: bottom;
       padding: 0.75rem;
       border-left: 1px solid #e7e7e7;
       line-height: 1;
+
       .datatable-header-cell-label {
         line-height: 24px;
         font-weight: 600;
       }
     }
+
     .datatable-header-cell:first-child {
       border-left: 0;
     }
   }
+
   .datatable-body {
     border-collapse: collapse;
+
     .datatable-body-row {
       vertical-align: top;
       border-top: 1px solid #e7e7e7;
       background-color: #f7f7f7;
+
       &.datatable-row-even {
         background-color: $content-block-bg-color;
       }
+
       &:hover {
         background-color: #eee;
       }
-      &.active {
+      // &.active {
         //background-color: blue;
         //font-weight: 600;
-      }
+      // }
```

Comment:
?

Reply:
it was complaining that it was "illegal" to have it empty:
```
&.active { }
```
 since the stuff inside it is commented

Reply:
ah ok ok